File: CBT_Backend/Models/Options.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace CBT_Backend.Models
{
    public class Options
    {
        public string? Id { get; set; }
        public string? Option_A { get; set; }
        public string? Option_B { get; set; }
        public string? Option_C { get; set; }
        public string? Option_D { get; set; }
        public string? Answer { get; set; }
        public DateTime Created_at { get; set; }
        public DateTime Updated_at { get; set; }
        public string? Assessment_Id { get; set; }
    }
}

File: cbt_frontpage/src/Components/Options.js
import React, { useState} from 'react'

function Options({Options}) {
    const [optionA, setOptionA] = useState('')
    const [optionB, setOptionB] = useState('')
    const [optionC, setOptionC] = useState('')
    const [optionD, setOptionD] = useState('')
    // console.log(Options.length);
    if(Options != null)
    {
        setOptionA(Options['Option_A'])
        setOptionB(Options['Option_B'])
        setOptionC(Options['Option_C'])
        setOptionD(Options['Option_D'])

        console.log(optionA);
        console.log(optionB);
        console.log(optionC);
        console.log(optionD);
    }
    
  return (
    <div>
        <p>{optionA}</p>
        <p>{optionB}</p>
        <p>{optionC}</p>
        <p>{optionD}</p>
    </div>
  )
}

export default Options
